lib/diagnostic.js

```diff
@@ -19,15 +19,15 @@
 //
 //     `true` or `false`
 //
-let access = null; // replace null with your response
+let access = false; // replace null with your response
```

Comment:
True.  Think of it as "child" scopes have access to "parent" scopes, but not vice vera.